frontends/web/src/components/aopp/aopp.tsx

```diff
@@ -34,85 +40,189 @@ interface SubscribedProps {
 type Props = SubscribedProps & AoppProps & TranslateProps;
 
 class Aopp extends Component<Props, State> {
-    private chooseAccount = (code: AccountCode) => {
-        aoppAPI.chooseAccount(code);
+    public readonly state: State = {
+        accountCode: '',
+    };
+
+    public componentDidMount() {
+        this.setAccountCodeDefault();
+    }
+
+    public componentDidUpdate(prevProps) {
+        if (this.props.aopp?.accounts !== prevProps.aopp?.accounts) {
+            this.setAccountCodeDefault();
+        }
+    }
+
+    private setAccountCodeDefault() {
+        const { aopp } = this.props;
+        if (aopp?.accounts && aopp?.accounts?.length) {
+            this.setState({ accountCode: aopp.accounts[0].code });
+        }
     }
 
+    private chooseAccount = (e: Event) => {
+        if (this.state.accountCode) {
+            aoppAPI.chooseAccount(this.state.accountCode);
+        }
+        e.preventDefault();
+    }
+
+    private Banner = ({ children }) => (<div className={styles.banner}>{children}</div>);
+    private Content = ({ children }) => (<div className={styles.content}>{children}</div>);
+    private Header = (props) => (
+        <header className={styles.header}>
+            <h1 className={styles.title}>{props.title}</h1>
+            {props.children}
+        </header>
+    );
+
```

Comment:
Please move them out of the class, as they don't need any class props/state/vars. Also add types if possible.

Reply:
Moved Header and Content to the fullscreen module and added types.

Kept Banner in this module, but moved out of the class.

PTAL if ok I'll squash into this commit
https://github.com/digitalbitbox/bitbox-wallet-app/pull/1373/commits/78068e45f1a922c5e4d63abf189b15a9a3371320

Reply:
:heavy_check_mark: